composer/utils/eval_client/mcli_eval_client.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""MCLI compatible eval client."""
+import logging
+import os
+from typing import Dict, List
+import time
+import numpy as np
+
+import mcli
+
+from composer.utils.eval_client.eval_client import EvalClient
+
+__all__ = ['MCLIEvalClient']
+log = logging.getLogger(__name__)
+
+
+class MCLIEvalClient(EvalClient):
+    """Utility for creating a client for and invoking an AWS Lambda through MCLI."""
+
+    def __init__(self, backoff: int = 3, num_retries: int = 5) -> None:
+        """Checks that the requisite environment variables are in the EvalClient.
+
+        There must be MOSAICML_PLATFORM to be on MCLI.
+        """
+        if os.environ.get('MOSAICML_PLATFORM', False) == False:
```

Comment:
This isn't really true right? You just need to have an api key or access token configured (which the mosaicml platform does for you). You could, say, run this locally or manually on a remote machine with mcli configured

It was slightly different for the MosaicMLLogger, because we also relied on the run name environment variable, which the platform sets